index.d.ts

```diff
@@ -219,6 +219,13 @@ export interface Options {
 	*/
 	readonly disableAnimations?: boolean;
 
+	/**
+	Disable [Ad Blocker](https://github.com/ghostery/adblocker/blob/master/packages/adblocker-puppeteer/README.md).
+
+	@default false
+	*/
+	readonly disableAdBlocker?: boolean;
```

Comment:
I don't like double-negatives. Better to do `blockAds` and have it on by default.

`disableAnimations` is different as it describes the action, not the option itself.